src/templates/app/trade/explanation.jsx

```diff
@@ -145,6 +145,12 @@ const Explanation = () => (
                 <p>{it.L('If you select [_1]"High Tick"[_2], you win the payout if the selected tick is the [_1]highest among the next five ticks[_2].', '<strong>', '</strong>')}</p>
                 <p>{it.L('If you select [_1]"Low Tick"[_2], you win the payout if the selected tick is the [_1]lowest among the next five ticks[_2].', '<strong>', '</strong>')}</p>
             </div>
+
+            <div id='winning_runs' className='invisible'>
+                <h3>{it.L('Winning the contract')}</h3>
+                <p>{it.L('If you select [_1]“Ups”[_2], you win the payout if consecutive tick(s) rise successively after the [_1]entry spot[_2].[_3]No payout if any tick falls or is equal to any of the previous ticks.', '<strong>', '</strong>', '<br />')}</p>
```

Comment:
can you change `“` and `”` to normal `"`?

Reply:
Yes.